site/components/grid/index.js

```diff
@@ -1,3 +1,8 @@
+/*
+* This component is ported from Website Next and is meant to be
+* used only by Jobs and Events components
+*/
+
```

Comment:
I'll review once again the components, and might as well get rid of these comments, or refine them somewhat.